### PR TITLE
Don't add spaces inside parens/etc

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

File: lib/Shower/Printer.hs [hunk 1]
```diff
@@ -18,20 +18,20 @@ instance Shower ShowerDoc where
 
 showerRecord' :: [(PP.Doc, PP.Doc)] -> PP.Doc
 showerRecord' fields =
-  PP.hsep [PP.lbrace, PP.nest 2 (showerFields fields), PP.rbrace]
+  mconcat [PP.lbrace, PP.nest 2 (showerFields fields), PP.rbrace]
```
Comment (int-index) [inline lib/Shower/Printer.hs]: PP.braces

File: lib/Shower/Printer.hs [hunk 2]
```diff
@@ -18,20 +18,20 @@ instance Shower ShowerDoc where
...
   where
     showerFields = PP.sep . PP.punctuate PP.comma . map showerField
     showerField (name, x) = PP.hang (name PP.<+> PP.equals) 2 x
 
 showerList' :: [PP.Doc] -> PP.Doc
 showerList' elements =
-  PP.hsep [PP.lbrack, PP.nest 2 (showerElements elements), PP.rbrack]
+  mconcat [PP.lbrack, PP.nest 2 (showerElements elements), PP.rbrack]
```
Comment (int-index) [inline lib/Shower/Printer.hs]: PP.brackets

File: lib/Shower/Printer.hs [hunk 3]
```diff
@@ -18,20 +18,20 @@ instance Shower ShowerDoc where
...
   where
     showerElements = PP.sep . PP.punctuate PP.comma
 
 showerTuple' :: [PP.Doc] -> PP.Doc
 showerTuple' elements =
-  PP.hsep [PP.lparen, PP.nest 2 (showerElements elements), PP.rparen]
```
Comment (int-index) [inline lib/Shower/Printer.hs]: PP.parens

**Comment by neongreen** [comment]
Fixed

**Pull request merged**